denonavr/foundation.py

```diff
@@ -535,6 +535,87 @@ async def async_power_off(self) -> None:
         else:
             await self.api.async_get_command(self.urls.command_power_standby)
 
+    async def async_cursor_up(self) -> None:
```

Comment:
Please call each of your new functions from `DenonAVR` class like in [this example](https://github.com/ol-iver/denonavr/blob/2f18b79ea9cda6ea5c6310300e3767d76e42ec2f/denonavr/denonavr.py#L667-L669). Otherwise the methods won't be publicly accessible.